package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "exist-documentation",
+  "version": "4.0.5-SNAPSHOT",
```

Comment:
should this value be hardcoded in here? or should it be generated ?

Reply:
see my comments in #151, like the readme if we process the version number via the pom, it ll work on the initial run and then be stuck with that version number. If we use `${version}` the file will be invalid. I m open to suggestions, since we're not publishing on npm, or including the file in the packaged xar, we could try to not run with a version number at all?

Reply:
see f02add20c8a403e52f81b41c14495334e8887933

Reply:
I would think it should be templated in from Maven and live in the src/main/resources folder or similar?